Scrabble.java

```diff
@@ -85,9 +107,17 @@ public static void playHand(String hand) {
 			// non-whitespace characters. Whitespace is either space characters, or  
 			// end-of-line characters.
 			String input = in.readString();
-			//// Replace the following break statement with code
-			//// that completes the hand playing loop
-			break;
+			if (MyString.subsetOf(".", input)) {
+				break;
+			}
+			if (isWordInDictionary(hand) && hand != "") {
+				score += wordScore(input);
+				hand = MyString.remove(hand, input);
+			}
+			else{
+				System.out.println(	"Invalid word. Try again.");
+			}

```

Comment:
not handled case subset input hand
example:
hand = "h e l l o", dictionary = [low] , input = "low" 
expected: no change
actual: l is removed